pkg/ibmvpc/routing_tables_analysis.go

```diff
@@ -59,13 +61,34 @@ func (ga *GlobalRTAnalyzer) GetRoutingPath(src vpcmodel.InternalNodeIntf, dest *
 		if err != nil {
 			return nil, err
 		}
-		res2, err := targetVPCAnalyzer.getIngressPath(tgwSource, dest)
+		targetVPC := ga.allConfigs.GetVPC(res.TargetVPC()).(*VPC)
+		destZone, _ := getZoneByIPBlock(dest, ga.allConfigs)
```

Comment:
because `dest` is not necessarily in the target VPC.
(for example in hub-n-spoke, there is an example where dest is in the enterprise vpc, (when looking to route from spoke to enterprise via transit), but the target vpc is the transit vpc).